scripts/install_dev_tools

```diff
@@ -173,42 +198,53 @@ else
     VENV="${venv_dir}"
 fi
 
+# Get the list of CLI tools to be installed from comma separated output
+readarray -td, cli_tools_arr <<<"$cli_tools"
+
 # Create download directory inside virtual env dir
 DWN_DIR=$(TMPDIR="${VENV}" mktemp -d -t "${TMP_DIR_PREFIX}XXXXX") || exit 2
 
 trap 'cleanup_and_exit' INT TERM EXIT
 
 # Helm install
-if ! [ -x "$(command -v "${VENV}/bin/helm")" ]; then
-    echo "Installing helm CLI to: ${VENV}/bin/helm"
-    download_file_from_url "${HELM_INSTALL_SCRIPT}" "${DWN_DIR}"
-    HELM_SCRIPT="${DWN_DIR}"/$(basename "${HELM_INSTALL_SCRIPT}")
-    install_helm "${HELM_SCRIPT}" "${VENV}/bin"
+if should_cli_be_installed "helm" "${cli_tools_arr[@]}"; then
+    if ! [ -x "$(command -v "${VENV}/bin/helm")" ]; then
```

Comment:
We could use:
```bash
should_cli_be_installed "helm" "${cli_tools_arr[@]}" && ! [ -x "$(command -v "${VENV}/bin/helm")" ]
```

But that's just a style thing.

Reply:
Changed.